app-wordpress/charts/wordpress/templates/pvc.yaml

```diff
@@ -1,23 +1,11 @@
-# {{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
-# apiVersion: v1
-# kind: PersistentVolumeClaim
-# metadata:
-#   name: {{ include "wordpress.fullname" . }}
-#   labels: {{- include "wordpress.labels" . | nindent 4 }}
-# spec:
-#   accessModes:
-#   {{- if not (empty .Values.persistence.accessModes) }}
-#   {{- range .Values.persistence.accessModes }}
-#     - {{ . | quote }}
-#   {{- end }}
-#   {{- else }}
-#     - {{ .Values.persistence.accessMode | quote }}
-#   {{- end }}
-#   resources:
-#     requests:
-#       storage: {{ .Values.persistence.size | quote }}
-#   {{- include "wordpress.storage.class" (dict "persistence" .Values.persistence "global" .Values.global) | nindent 2 }}
-#   {{- if .Values.persistence.dataSource }}
-#   dataSource: {{- include "wordpress.tplvalues.render" (dict "value" .Values.persistence.dataSource "context" $) | nindent 4 }}
-#   {{- end }}
-# {{- end }}
\ No newline at end of file
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: wordpress-pvc
+  labels: {{- include "wordpress.labels" . | nindent 4 }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
```

Comment:
We may need to grow this in future as the wordpress volume is likely to fill up much faster than the mariadb.  Not a problem for now though we can expand in future 👍🏻